state-chain/runtime/src/runtime_apis.rs

```diff
@@ -57,7 +57,9 @@ pub struct RuntimeApiAccountInfoV2 {
 
 #[derive(Encode, Decode, Eq, PartialEq, TypeInfo, Serialize, Deserialize)]
 pub struct ChainAsset {
+	#[serde(flatten)]
```

Comment:
I'm not sure this will do anything in this case, same with the asset below.

Reply:
I think this type isn't used now?

Reply:
Good catch 🕵️